source/LogbookPanel.cpp

```diff
@@ -167,6 +171,16 @@ void LogbookPanel::Draw()
 			pos.Y() += wrap.Height() + GAP;
 		}
 	}
+	else if(reputationsSelected)
+	{
+		for(const pair<string, string> &item : reputations)
+		{
+			font.Draw(item.first, pos + textOffset, bright);
+			DisplayText text(item.second, {static_cast<int>(TEXT_WIDTH - 2. * PAD), Alignment::RIGHT, Truncate::BACK});
```

Comment:
Is this too long? Not sure

Reply:
My count has it at exactly 120 characters.